.gitignore

```diff
@@ -348,3 +348,5 @@ fabric.properties
 .idea/caches/build_file_checksums.ser
 
 .temp
+
+/result
```

Comment:
the /result exclusion is already present at the top of the file

Reply:
Done